AppendBlobReader/tests/test_appendblobreader.py

```diff
@@ -54,28 +53,95 @@ def test_02_resource_count(self):
         self.check_resource_count(expected_resource_count)
     
     def test_03_insert_chunks(self):
+        
         self.logger.info("inserting mock data in BlobStorage")
         self.upload_file_chunks_using_append_blobs()
 
         time.sleep(600)
 
         app_insights = self.get_resource('Microsoft.Insights/components')
-        captured_output = self.fetchlogs(app_insights.name)
-        # self.logger.info("captured_output:..\n\n", captured_output)
-        successful_sent_message = "All chunks successfully sent to sumo."
-        self.assertTrue(self.filter_logs(captured_output, 'message', successful_sent_message),
-                        "No success message found in azure function logs")
 
-        # end_message = "Offset is already at the end, Exit now!"
-        # self.assertTrue(self.filter_logs(captured_output, 'message', end_message),
-        #                 "No success message found in azure function logs")
+        # Azure function: BlobTaskProducer
+        azurefunction = "BlobTaskProducer"
+        captured_output = self.fetchlogs(app_insights.name, azurefunction)
+
+        message = "Append blob scenario create just an entry RowKey:"
+        self.assertTrue(self.filter_logs(captured_output, 'message', message),
+                        f"No success '{message}' message found in '{azurefunction}' function logs")
```

Comment:
No '{message}' log line found in '{azurefunction}' function logs

Reply:
Updated